pcs/lib/corosync/qdevice_client.py

```diff
@@ -15,9 +13,7 @@ def get_status_text(runner: CommandRunner, verbose: bool = False) -> str:
     verbose -- get more detailed output
     """
     cmd = [
-        os.path.join(
-            settings.corosync_qdevice_binaries, "corosync-qdevice-tool"
-        ),
+        settings.corosync_qdevice_tool_exec,
         "-s",
```

Comment:
```suggestion
        "-s"
```
This will make black to put the whole list in one line. It's a formatting change, but it's small enough for us to take it.